apps/web/src/state/app/settings.ts

```diff
@@ -18,3 +18,15 @@ export const setPrivacy = async ({ hideBalance }: { hideBalance: boolean }) => {
 		return doc;
 	});
 };
+
+export const setPathname = async ({
+	currentScreen,
+}: {
+	currentScreen: string;
+}) => {
+	await modules.storage.upsert<SettingDocument>(id, async (doc) => {
+		doc.config.currentScreen = currentScreen;
+
+		return doc;
+	});
+};
```

Comment:
I realized that there's a parameter mistake in my previous `setPrivacy` that it does not need to destruct `hideBalance` there. I will explain more in chat.